Cargo.toml

```diff
@@ -52,3 +52,6 @@ array_tool = "1.0.3"
 reqwest = {version="0.11.4"}
 url = {version="2.2.2"}
 console = "0.15.0"
+
+[dev-dependencies]
+tempfile = "3.3.0"
```

Comment:
This crate provides a tmpdir that is automatically deleted.

ref:
https://docs.rs/tempfile/3.3.0/tempfile/fn.tempdir.html

---

if the destructor is not moved, leaking will occur.

ref:
https://docs.rs/tempfile/3.3.0/tempfile/struct.TempDir.html#resource-leaking